include/lib/cpus/aarch64/cortex_a53.h

```diff
@@ -41,6 +41,15 @@
 
 #define CPUECTLR_SMP_BIT		(1 << 6)
 
```

Comment:
> Also, it would be good to add defines for the Advanced SIMD and Floating-point retention control bits for completeness.

Where can I get a description of these values? I checked the "Advance SIMD and Floation Point" section in the A57r1p3 TRM.